articles/mysql/flexible-server/how-to-deploy-on-azure-free-account.md

```diff
@@ -1,4 +1,4 @@
----
+                           ---
```

Comment:
```suggestion
---
```

Reply:
@ShannonLeavitt - Thanks for the suggestion.